doc/building_sdk.md

```diff
@@ -0,0 +1,30 @@
+#### Building SDK
```

Comment:
is this linked from anywhere? Should it be?

Reply:
Yes.  I had done that and not pushed.

Reply:
Not yet.

Reply:
addressed